src/qt/bitcoingui.cpp

```diff
@@ -350,6 +350,11 @@ void BitcoinGUI::createActions()
     showHelpMessageAction->setMenuRole(QAction::NoRole);
     showHelpMessageAction->setStatusTip(tr("Show the %1 help message to get a list with possible Bitcoin command-line options").arg(PACKAGE_NAME));
 
+    m_mask_values_action = new QAction(tr("&Mask values"), this);
+    m_mask_values_action->setShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_M));
+    m_mask_values_action->setStatusTip(tr("Cloak values on Overview tab"));
```

Comment:
- s/values on/the values in the|all values in the/ 

- I'd suggest keeping the verb consistent everywhere, e.g. s/Cloak/Mask/

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16432#issuecomment-633063508).